src/data/mystery-encounters/encounters/uncommon-breed-encounter.ts

```diff
@@ -210,9 +210,7 @@ export const UncommonBreedEncounter: MysteryEncounter = MysteryEncounterBuilder.
 
         // Remove 4 random berries from player's party
         // Get all player berry items, remove from party, and store reference
-        const berryItems: BerryModifier[] = globalScene.findModifiers(
-          (m) => m instanceof BerryModifier,
-        ) as BerryModifier[];
+        const berryItems: BerryModifier[] = globalScene.findModifiers((m) => m.isBerryModifier()) as BerryModifier[];
```

Comment:
```suggestion
        const berryItems: BerryModifier[] = globalScene.findModifiers((m) => m.isBerryModifier());
```
`const x: type = z as type;` :pikasquint:

Reply:
It's necessary because `findModifiers` doesn't return flexible types

Reply:
Ah, I guess the unnecessary part is the `: type` then.